chia/simulator/setup_nodes.py

```diff
@@ -283,10 +283,10 @@ async def setup_simulators_and_wallets_inner(
             )
         )  # block tools modifies constants
         sim = setup_full_node(
-            bt_tools[index].constants,
-            bt_tools[index].config["self_hostname"],
-            db_name,
```

Comment:
https://github.com/Chia-Network/chia-blockchain/blob/d1a4c579f4dd2b058fcd46f2249a876a60b1dec5/chia/simulator/setup_services.py#L80-L83

Reply:
assuming this can be resolved.